lib/rules/no-useless-escape.ts

```diff
@@ -46,6 +46,8 @@ const REGEX_ESCAPES = new Set([
     CP_CLOSING_PAREN, // )
 ])
 
+const POTENTIAL_ESCAPE_SEQUENCE = new Set("uxkpP".split(""))
```

Comment:
It is the same without `.split("")`.

```suggestion
const POTENTIAL_ESCAPE_SEQUENCE = new Set("uxkpP")
```